PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml

```diff
@@ -19,6 +19,19 @@
   <change-notes>
     <![CDATA[
     <html>
+      <h3>[3.12.0]</h3>
+      <ul>
+        <li>Updated to list both Windows and Linux web apps in web apps explorer.</li>
+        <li>Added support to choose a Linux web app to deploy the application.</li>
+        <li>Added support to create a new Linux web app to deploy the application.</li>
+        <li>Added support to show all the deployment slots for a web app.</li>
+        <li>Refactored web apps icons.</li>
+        <li>Refactored the use existing web apps UI when users click 'Run on Web App'.</li>
+        <li>Updated to list all app service plans of the selected subscription for users to choose to create a new web app.</li>
+        <li>Updated to use zipDeploy to upload none war files.</li>
+        <li>Updated to always deploy the web.config file when deploying jar artifacts to Windows web apps.</li>
```

Comment:
- "Web App". 
- The saying is a little awkward for "deploy config file when deploying jar file". Actually it **deploys the application** by **uploading** config files together with artifacts. So I would suggest to use something like "Always updates/uploads the web.config file together with the .jar artifact when deploying to Web App (Windows)."

Reply:
How about `Always uploads the web.config file together with the .jar artifact when deploying to Windows web apps.`

`Windows/Linux web apps` looks better to me.